Sources/BraveWallet/Crypto/Stores/NFTStore.swift

```diff
@@ -121,14 +178,57 @@ public class NFTStore: ObservableObject {
       let filters = self.filters
       let selectedAccounts = filters.accounts.filter(\.isSelected).map(\.model)
       let selectedNetworks = filters.networks.filter(\.isSelected).map(\.model)
-      let allVisibleUserAssets = assetManager.getAllVisibleAssetsInNetworkAssets(networks: selectedNetworks)
-      self.userVisibleNFTs = buildAssetViewModels(allVisibleUserAssets: allVisibleUserAssets)
-        .optionallyFilterUnownedNFTs(
-          isHidingUnownedNFTs: filters.isHidingUnownedNFTs,
-          selectedAccounts: selectedAccounts
-        )
       
-      let allTokens = allVisibleUserAssets.flatMap(\.tokens)
+      // user visible assets
+      let userVisibleAssets = assetManager.getAllUserAssetsInNetworkAssets(networks: selectedNetworks, visible: true)
+      
+      // user invisible assets
+      let userInvisibleAssets = assetManager.getAllUserAssetsInNetworkAssets(networks: selectedNetworks, visible: false)
+      
+      // all spam NFTs marked by user
+      let allUserMarkedSpamNFTs = assetManager.getAllUserNFTs(networks: selectedNetworks, spamStatus: true)
+      
+      // all spam NFTs marked by SimpleHash
+      let simpleHashSpamNFTs = await walletService.simpleHashSpamNFTs(for: selectedAccounts, on: selectedNetworks)
```

Comment:
Probably okay to remove the whitespace here between each declaration as they have comments separating them also?